app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -225,7 +226,12 @@ class Exposure extends React.Component {
       .then(function() {
         // No validation issues? Invoke callback (move to next step)
         self.setState({ errors: {} }, async () => {
-          if (self.state.current.patient.jurisdiction_id !== self.state.originalJurisdictionId) {
+          if (parseInt(self.state.current.patient.jurisdiction_id) !== self.state.originalJurisdictionId) {
```

Comment:
I think with the above change I mentioned these changes shouldn't be necessary at all because the `parseInt` won't be needed, and this should evaluate to false, right?

Reply:
Correct, with the above change we don't need this

Reply:
So I see in your change to address this here you still include the check below this line. That shouldn't be necessary anymore either as it's not possible for both self.state.current.patient.jurisdiction_id !== self.state.originalJurisdictionId to be true and for self.state.current.patient.jurisdiction_id === self.state.selected_jurisdiction to be true.

Reply:
So I see in your change to address this here you still include the check below this line. That shouldn’t be necessary anymore either as I don't think we need `self.state.selected_jurisdiction` at all anymore. Right?

Reply:
The `selected_jurisdiction` is to fix the following:
1. Change the jurisdiction from `State 1` to `State 1, County 1` and click next then click ok to go to the next page
2. Click previous to go back, click next - you will be asked to save your changes to jurisdiction from `State 1` to `State 1, County 1` even though you hadn't changed that field again

Reply:
Ah right, yep that's fine then. Thanks!